assets/javascripts/discourse/components/query-result.js.es6

```diff
@@ -35,6 +35,16 @@ const QueryResultComponent = Ember.Component.extend({
     return this.get('content.columns').length;
   }.property('content.columns.length'),
 
+  resultCount: function() {
+    if (this.get('content.result_count') === 1) {
```

Comment:
Helllo @SamSaffron :wave: 
This logic works correctly but @techAPJ mentioned that a new Data Explorer SiteSetting (maybe hidden) for result limit could be useful.
Currently, the 250-limit is hardcoded in two files. (plugin.rb, query-result.js) 
What do you think is better?